Iterations over an Innovation System

malcata opened the pull request:
Iterations over an Innovation System
=================================================

* Speaker   : **João Malcata**
* Available : **first day, second day**
* Length    : **60 minutes**
* Language  : **English**

Description
-----------

The journey of a startup studio iterating over an innovation system to ignite the launch of six ventures. From idea generation to the business model in an industry agnostic setting. Lessons learned along the way are shared and expected to broaden perspectives leading to the improvement of innovation initiatives in general.

Keywords: Entrepreneurship, Innovation, Processes

Speaker Bio
-----------

Dad, software engineer, entrepreneur, telco, web, cybersecurity and OSS. 20 years building tech products, teams, and processes. Nowadays, you can find him chasing some new product in a new industry, combining the wisdom of the crowd with machine learning.

Links
-----


* Blog: https://medium.com/@malcata
* Company: http://intersection.ventures
* GitHub: https://github.com/malcata
* Photo: https://raw.githubusercontent.com/PixelsCamp/talks/master/img/joao_malcata.jpg

Extra Information
-----------------

[From zero to Continuous Delivery in 6 months (Pixels Camp 2019)](https://github.com/PixelsCamp/talks/blob/master/2019/from-zero-to-continuous-delivery-in-6-months_joao-malcata.md)
[A brief and personal history of the software development-process (Pixels Camp 2017)](https://github.com/PixelsCamp/talks/blob/master/2017/a-brief-and-personal-history-of-the-software-development-process_joao-malcata.md)

carlosefr commented:
Hi, your proposal was submitted after the deadline. This doesn't mean it won't be considered, but means it will be considered only after the ones that were submitted before/on January 20.

Also, your description and bio is formatted as bold. Please update the PR description and the `.md` file to remove it.

malcata commented:
@carlosefr should be fixed now. Somehow when I created a second talk while trying to make it independent from the first one I have removed the .md file. Fixed.
